packages/web-components/fast-foundation/src/checkbox/checkbox.ts

```diff
@@ -69,10 +55,6 @@ export class FASTCheckbox extends FormAssociatedCheckbox {
      * @internal
      */
     public keypressHandler = (e: KeyboardEvent): void => {
-        if (this.readOnly) {
```

Comment:
Shouldn't this be updated to be `if (this.disabled)` in order to match the `clickHandler`?

Reply:
Opening this back up as I do think that this is a bug if the event is emitted even when disabled here...likely should be a noop...should we handle here or in a new fix? Considering we've already condensed another existing method as part of this, while I prefer to not batch commits, this may be a reasonable addition. Thoughts @yinonov?

Reply:
of course! done